.github/CODEOWNERS

```diff
@@ -44,72 +43,42 @@
+/packages/selected-network-controller     @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers @MetaMask/metamask-assets
+/packages/profile-sync-controller         @MetaMask/notifications @MetaMask/identity
+/packages/remote-feature-flag-controller  @MetaMask/extension-platform @MetaMask/mobile-platform
+
 ## Wallet Framework Team
+**/CHANGELOG.md                        @MetaMask/wallet-framework-engineers
+**/package.json                        @MetaMask/wallet-framework-engineers
+**/tsconfig.*.json                     @MetaMask/wallet-framework-engineers
+**/tsconfig.json                       @MetaMask/wallet-framework-engineers
+**/typedoc.json                        @MetaMask/wallet-framework-engineers
+/*.js                                  @MetaMask/wallet-framework-engineers
+/*.json                                @MetaMask/wallet-framework-engineers
+/*.mjs                                 @MetaMask/wallet-framework-engineers
+/.*                                    @MetaMask/wallet-framework-engineers
+/README.md                             @MetaMask/wallet-framework-engineers
+/docs                                  @MetaMask/wallet-framework-engineers
+/examples                              @MetaMask/wallet-framework-engineers
 /packages/base-controller              @MetaMask/wallet-framework-engineers
 /packages/build-utils                  @MetaMask/wallet-framework-engineers
 /packages/composable-controller        @MetaMask/wallet-framework-engineers
 /packages/controller-utils             @MetaMask/wallet-framework-engineers
 /packages/polling-controller           @MetaMask/wallet-framework-engineers
 /packages/preferences-controller       @MetaMask/wallet-framework-engineers
-
-## Wallet UX Team
-/packages/announcement-controller      @MetaMask/wallet-ux
-
-## Joint team ownership
-/packages/eth-json-rpc-provider        @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
-/packages/json-rpc-engine              @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
-/packages/json-rpc-middleware-stream   @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
-/packages/network-controller           @MetaMask/wallet-framework-engineers @MetaMask/metamask-assets
-/packages/permission-controller        @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers @MetaMask/snaps-devs
-/packages/permission-log-controller    @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
-/packages/selected-network-controller  @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers @MetaMask/metamask-assets
-/packages/profile-sync-controller      @MetaMask/notifications @MetaMask/identity
-
-## Package Release related
-/packages/accounts-controller/package.json        @MetaMask/accounts-engineers @MetaMask/wallet-framework-engineers
```

Comment:
I was trying to avoid from having to do this 🤔 What I've done isn't right. Maybe there is another way.